generic3g/specs/StateItemAspect.F90

```diff
@@ -166,7 +166,7 @@ logical function can_connect_to(src, dst)
            end if
         case (1)
            can_connect_to = .true.
-        case (2)
```

Comment:
The only difference in this file is the identation, and I like the older version better

Reply:
Fixed.